ktor-http/jvm/test/io/ktor/tests/http/BitmaskCodecsBenchmarkTest.kt

```
@@ -0,0 +1,513 @@
+    private fun buildReport(
+        timings: Map<String, TimingResult>,
+        allocations: Map<String, Pair<Double?, Double?>>
+    ): String = buildString {
+        appendLine()
+        appendLine("=== BitmaskCodecsBenchmarkTest (${java.time.LocalDateTime.now()}) ===")
+        appendLine(
+            "NOTE: this is a same-JVM, in-process A/B micro-harness (not JMH) -- " +
+                "no fork isolation, treat numbers as indicative, not authoritative."
+        )
+        appendLine()
+        appendLine(String.format("%-18s %14s %14s %10s", "operation", "old ns/op", "new ns/op", "speedup"))
```

detekt (1.23.8)

[warning] String.format("%-18s %14s %14s %10s", "operation", "old ns/op", "new ns/op", "speedup") uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)